gdx/src/com/badlogic/gdx/maps/tiled/AtlasTmxMapLoader.java

```diff
@@ -27,40 +25,40 @@
 import com.badlogic.gdx.graphics.Texture.TextureFilter;
 import com.badlogic.gdx.graphics.g2d.TextureAtlas;
 import com.badlogic.gdx.graphics.g2d.TextureAtlas.AtlasRegion;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.badlogic.gdx.maps.MapLayers;
 import com.badlogic.gdx.maps.MapProperties;
 import com.badlogic.gdx.maps.tiled.tiles.AnimatedTiledMapTile;
 import com.badlogic.gdx.maps.tiled.tiles.StaticTiledMapTile;
-import com.badlogic.gdx.utils.Array;
-import com.badlogic.gdx.utils.GdxRuntimeException;
-import com.badlogic.gdx.utils.IntArray;
-import com.badlogic.gdx.utils.ObjectMap;
-import com.badlogic.gdx.utils.SerializationException;
+import com.badlogic.gdx.utils.*;
 import com.badlogic.gdx.utils.XmlReader.Element;
 
+import java.io.IOException;
+
 /** A TiledMap Loader which loads tiles from a TextureAtlas instead of separate images.
- * 
+ *
  * It requires a map-level property called 'atlas' with its value being the relative path to the TextureAtlas. The atlas must have
  * in it indexed regions named after the tilesets used in the map. The indexes shall be local to the tileset (not the global id).
  * Strip whitespace and rotation should not be used when creating the atlas.
- * 
+ *
  * @author Justin Shapcott
  * @author Manuel Bua */
 public class AtlasTmxMapLoader extends BaseTmxMapLoader<AtlasTmxMapLoader.AtlasTiledMapLoaderParameters> {
 
 	public static class AtlasTiledMapLoaderParameters extends BaseTmxMapLoader.Parameters {
-		/** force texture filters? **/
-		public boolean forceTextureFilters = false;
```

Comment:
i think this is a non implemented feature and should stay here for now because client code may use this variable with their AtlasTmxMapLoader override.

3 possible solutions : 

* don't remove this variable for now (recommended IMO)
* implement  forceTextureFilters feature
* let it removed but add it in API changes file